packages/google_maps_flutter/google_maps_flutter/example/lib/main.dart

```diff
@@ -66,5 +68,11 @@ class MapsDemo extends StatelessWidget {
 }
 
 void main() {
+  WidgetsFlutterBinding.ensureInitialized();
```

Comment:
Why is this necessary? Setting that flag doesn't send a message, does it?

Reply:
You're right. Force of habit. removed